crates/config/src/lib.rs

```diff
@@ -196,6 +196,8 @@ pub struct Config {
     pub libraries: Vec<String>,
     /// whether to enable cache
     pub cache: bool,
+    /// whether to enable preprocessed cache for tests
+    pub optimize_tests: bool,
```

Comment:
Should we also add it to https://github.com/foundry-rs/foundry/blob/master/crates/cli/src/opts/build/core.rs ? That would make it easier to toggle it on / off from the CLI

Reply:
in the future we could add a `--no-cache-tests` mirroring the `--no-cache` build flag once we have enabled it by default